docs/troubleshoot/cli/known-cli.md

```diff
@@ -155,7 +154,7 @@ You receive the error message `npm ERR! Cannot read property 'pause' of undefine
 This behavior is due to a problem with Node Package Manager (npm). If
 you encounter this problem, revert to a previous version of npm that
 does not contain this defect. To revert to a previous version of npm,
-issue the following command:
+issue the following command: **[is the version correct? if not, what should it be?]**
```

Comment:
@adam-wolfe - please respond

Reply:
This is a really good question. Would need confirmation from engineering, but I think we could just delete the `## npm install -g command fails due to npm ERR! Cannot read property 'pause' of undefined error` section. I don't think we would want to tell people to install npm 5.3.0, which is very old.

Reply:
Can confirm that this issue has been resolved: https://github.com/npm/npm/issues/18520

We shouldn't need this info anymore as Adam suggested, npm v5 is old and we don't support it anymore.